pkgs/dartpad_ui/lib/main.dart

```diff
@@ -59,6 +59,10 @@ class _DartPadAppState extends State<DartPadApp> {
         path: '/',
         builder: _homePageBuilder,
       ),
+      GoRoute(
+        path: '/flutter',
+        builder: _homePageBuilder,
+      ),
```

Comment:
Since this ends up building the same page, what do you think about instead using a redirect that specifies a sample? Perhaps the [counter sample](https://dartpad.dev/?sample=counter) at `dartpad.dev/?sample=counter`, as that is similar to the default `flutter` tool app.

This could be a redirect in the [`firebase.json`](https://github.com/dart-lang/dart-pad/blob/main/pkgs/dartpad_ui/firebase.json) or a [GoRouter redirect](https://pub.dev/documentation/go_router/latest/go_router/RouteBase/redirect.html) if needed.

Or if you think a separate path is better, that's fine too! I'm curious what you think :)

Reply:
egarding the redirect, I thought about it while coding, but I had some concerns about readability. It’s not a big deal, though.